app/presenters/document/web_resource_presenter.rb

```diff
@@ -233,18 +259,26 @@ def downloadable?
           (media_type == 'video' && mime_type == 'text/plain; charset=utf-8')
         false
       else
-        @record_presenter.has_views.include?(url) || for_edm_is_shown_by?
+        for_has_view? || for_edm_is_shown_by?
       end
     end
 
     def for_edm_object?
-      @record_presenter.edm_object == url
+      url == @record_presenter.edm_object
+    end
+
+    def for_edm_is_shown_at?
```

Comment:
this isn't being called, is there a dynamically constructed method call or for potential future use?

Reply:
Nope, you're right, it's redundant.